server/app.js

```diff
@@ -5,32 +5,37 @@ var port = process.env.PORT || 3000;
 app.use(express.static(path.join(__dirname, "..", "build")));
 app.use(express.static("public"));
 
-var originBlacklist = parseEnvList(process.env.CORSANYWHERE_BLACKLIST);
-var originWhitelist = parseEnvList(process.env.CORSANYWHERE_WHITELIST);
-function parseEnvList(env) {
-  if (!env) {
-    return [];
-  }
-  return env.split(",");
-}
-
-// Set up rate-limiting to avoid abuse of the public CORS Anywhere server.
-var checkRateLimit = require("./lib/rate-limit")(
-  process.env.CORSANYWHERE_RATELIMIT
-);
-
-var cors_proxy = require("./lib/cors-anywhere");
-
-const proxy = cors_proxy.createServer({
-  originWhitelist: originWhitelist,
-  originBlacklist: originBlacklist,
-  requireHeaders: [], // Do not require any headers.
-  removeHeaders: [], // Do not remove any headers.
-});
+/* don't need to do CORS chekcking since we're deploying backend seperately
```

Comment:
Got it! Those links helped clear it up. So instead of relying on the cors functionality built into the fetch api we're just doing it ourselves using the linked repo to do it.